applicationdevelopment/assets/css/style.css

```diff
@@ -0,0 +1,120 @@
+body {
+	font-family: Myriad Pro;
+	color: #666666;
+	padding: 0;
+	font-size: 14px;
+}
+* {
+	box-sizing: border-box;
+}
+.clearfix:after {
+	visibility: hidden;
+	display: block;
+	font-size: 0;
+	content: " ";
+	clear: both;
+	height: 0;
+}
+* html .clearfix             { zoom: 1; } /* IE6 */
+*:first-child+html .clearfix { zoom: 1; } /* IE7 */
+
+ul,
+ol {
+  margin: 0;
+  padding: 0;	
+}
+a {
+	color: #13a0f8;
+}
+table {
+	border-spacing: 0;
+	border-collapse: collapse;
+}
+
+.wrapper {
+	width: 980px;
+	margin: auto;
+	background: #c2c2c2;
+}
+.tab-navi ul {
+	list-style: none;
+}
+.tab-navi ul > li {
+	float: left;
+	background: url('../images/navi-devider.png') top right no-repeat;
+
+}
+.tab-navi ul > li > a {
+	color: #000000;
+    display: block;
+    padding: 8px 20px;
+    text-shadow: 0 1px 0 #e5e5e5;
+    text-decoration: none;
+    background: linear-gradient(rgba(0, 0, 0, 0), #afafaf);
+    transition: all 0.4s;
+} 
+.tab-navi ul > li.active > a {
+	background: linear-gradient(#ffffff, #e5e5e5);
+	cursor: default;
+}
+.tab-navi ul > li:not(.active) > a:hover {
```

Comment:
active class gets applied only on one element at a time. It doesn't require to find li not having .active instead use normal selector and ovverride styles in .active